packages/react-reconciler/src/ReactFiberNewContext.js

```diff
@@ -25,13 +25,15 @@ import {createCursor, push, pop} from './ReactFiberStack';
 const providerCursor: StackCursor<Fiber | null> = createCursor(null);
 const valueCursor: StackCursor<mixed> = createCursor(null);
 const changedBitsCursor: StackCursor<number> = createCursor(0);
-const rendererSigilCursor: StackCursor<Object | null> = createCursor(null);
-const rendererSigilCursor2: StackCursor<Object | null> = createCursor(null);
 
 let rendererSigil;
+let rendererSigilCursor: StackCursor<Object | null>;
+let rendererSigilCursor2: StackCursor<Object | null>;
```

Comment:
Why does Flow pass if the whole thing might be uninitialized? Seems suspicious.

Reply:
Suspicious indeed https://flow.org/try/#0DYUwLgBAHgXA3gQxgZzAJwJYDsDmBfAbgCgBjAey1QgBEBRANRgCMyzQEsBedAVxGIwAzCAAo69AJQQ4RCNAidpSAEQBGAEwBmZYSJ4iQ0eKky55SmxAA6YGRwioVhBOJ4gA